pkg/cluster/cluster.go

```diff
@@ -1006,3 +1059,10 @@ func (c *Cluster) deletePatroniClusterConfigMaps() error {
 
 	return c.deleteClusterObject(get, deleteConfigMapFn, "configmap")
 }
+
+func (c *Cluster) deleteLogicalBackupJob() error {
+
+	c.logger.Debug("removing the logical backup job")
```

Comment:
As you mentioned before, if we'll delete a cronjob, a corresponding jobs will be also stopped. It means in turn that the backup would be interrupted, which in fact is something that can be important (e.g. if it takes long time). By itself it's ok, but we need to make this information visible, and since if I recall correctly we do not show `Debug` log messages by default, I suggest to raise it to something visible.

Reply:
we do log Debugf by default though